### PR TITLE
Keep a CHANGELOG

### PR DESCRIPTION
**Pull request by fiveop**
See nix-rust/nix#363

### CODE REVIEW

File: text/0000-use-changelog.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Change logs are a common and convenient way to document API changes in a single
+location. Being able to easily find notes on breaking changes is particularly
+useful.
+
+# Detailed design
+[design]: #detailed-design
+
+We follow the conventions laid out in [Keep A CHANGELOG](https://github.com/olivierlacan/keep-a-changelog/tree/18adb5f5be7a898d046f6a4acb93e39dcf40c4ad).
+
```
Comment (posborne) [inline text/0000-use-changelog.md:22]: Might be useful to include the following:
- Guidlines for contributors related to updating the changelog (probably encourage it but make it a soft requirement)
- Note responsibility for maintainer to update CHANGELOG before release including creating a new unreleased section.

**Comment by posborne** [comment]
Looks good generally.

**Comment by fiveop** [comment]
Thank you, for the feedback. I already added, that we create an initial CHANGELOG.md.

**Comment by kamalmarhubi** [comment]
Sorry for the general slowness lately!

File: text/0000-use-changelog.md (hunk 1 above)
Comment (kamalmarhubi) [inline text/0000-use-changelog.md:22]: > Guidlines for contributors related to updating the changelog (probably encourage it but make it a soft requirement)

:+1:  to that. We don't want it to block people from contributing, so maintainers can always fix it up later.

**Comment by kamalmarhubi** [comment]
Yeah I like this.

**Comment by fiveop** [comment]
Given, that I addressed everything @posborne brought up, and @kamalmarhubi did not have any concerns, we (someone) should merge this.

**Comment by posborne** [comment]
@homu r+

**Comment by posborne** [comment]
Looks like @kamalmarhubi will have to hit Merge.

**Comment by kamalmarhubi** [comment]
Will have to tinker with permissions...

**Pull request merged**